src/libraries/System.Security.Principal.Windows/src/Resources/Strings.resx

```diff
@@ -81,9 +81,6 @@
   <data name="ArgumentOutOfRange_ArrayTooSmall" xml:space="preserve">
     <value>Destination array is not long enough to copy all the required data. Check array length and offset.</value>
   </data>
-  <data name="ArgumentOutOfRange_NeedNonNegNum" xml:space="preserve">
-    <value>Non-negative number required.</value>
-  </data>
```

Comment:
Do we have any tooling to detect unused strings, or was all of this manual?

Reply:
The resx changes were manual.